### DISCUSSION TITLE
Can deepmd distinguish between O in the bulk phase and O in the gas?

### DISCUSSION THREAD

**Discussion by XuFanffei** [deepmodeling/deepmd-kit#3807] [Q&A]
### Summary

Can deepmd distinguish between O in the bulk phase and O in the gas?

### DeePMD-kit Version

2.1.5

### Backend and its version

TensorFlow v2.15.0

### Details

If possible, is it possible to classify O in the bulk phase and O in the gas, such as using O1 and O2 as different descriptors to describe their surrounding environment

**Comment by wanghan-iapcm** [comment]
They are of the same chemical type. the phases are distinguished by the coordinates of neighbor atoms.
